firefox-ios/l10n-screenshots.sh

```diff
@@ -34,19 +34,20 @@ for lang in $LOCALES; do
     echo "$(date) Snapshotting $lang"
     mkdir "l10n-screenshots/$lang"
     fastlane snapshot --project firefox-ios/Client.xcodeproj --scheme L10nSnapshotTests \
-        --number_of_retries 0 \
+        --number_of_retries 3 \
         --skip_open_summary \
         --xcargs "-maximum-parallel-testing-workers 2" \
         --derived_data_path l10n-screenshots-dd \
         --ios_version "18.2" \
         --erase_simulator --localize_simulator \
         --devices "iPhone 16" --languages "$lang" \
         --output_directory "l10n-screenshots/$lang" \
+        --xcodebuild_formatter xcbeautify \
```

Comment:
The default formatter on Bitrise is now `xcbeautify`. I have intermittent issues on `xcpretty` currently with the latest `fastlane`. Let me ensure that I use the same formatter locally.